### ISSUE TITLE
Make some craftables dual-profession

### ISSUE THREAD

**Issue by CraftsmasterKubilai** [Tirocupidus/TheExiledRPOverhaul#150]
As per Discord chat, players would like some items to be craftable under multiple profession lines, specifically:

1. Furnaces should be craftable by BOTH Builder 1 and Smith 1 (since they have direct utility for both lines).

2. Banners should be craftable by BOTH Weaver and Set Priest (since weaver just makes sense and the IG item has a snake on it, making it appropriate for Settites).

3. Last, Armorers should ALSO be able to craft metal ingots in furnaces (like Smiths) because of their dependency on that material for their own projects (this may already be how Professions work - this ticket is being written by a Smith so I'm not 100% sure - if so disregard/delete this acceptance criteria). 

There may be others as well - if so they will be added.

As per Tiro, adding recipes to multiple professions is do-able by modifying TERPO.

**Comment by AlexandriaWolf** [comment]
I completely agree with this. Nothing is more frustrating than going to make some things...and realizing the person you need to "push the button" on some component isn't online.  I'd add that builders need to be able to make what they need for their craft as well. Why as a T2 builder can I not make shaped wood when it's a necessary ingredient to make all the T2 pieces?

**Comment by CraftsmasterKubilai** [comment]
According to Mods chat in Discord, this (and other) problems SHOULD be fixed with the new 'modular' Professions system that's currently being worked on. Don't know how (I'm not working on that project), but I'm looking forward to what they're cooking up...
